src/command/dev/next/mod.rs

```diff
@@ -90,7 +94,11 @@ impl Dev {
             .resolve_federation_version(
                 &client_config,
                 make_fetch_remote_subgraph,
-                self.opts.supergraph_opts.federation_version.clone(),
+                self.opts
+                    .supergraph_opts
+                    .federation_version
+                    .clone()
+                    .or(Some(LatestFedTwo)),
```

Comment:
Adding this here, as otherwise it seems to default to the version on the subgraphs and from the test plan that seems to not be what we want. Happy to change if that is the case though